en/drafts/originals/understanding-creating-word-embeddings.md

```diff
@@ -441,71 +424,74 @@ for i in range(len(model_list)):
+- Ryan Heuser's [Word Vectors in the Eighteenth Century](https://ryanheuser.org/word-vectors/) walks through a research project using word vector models to understand eighteenth-century conceptions of originality.
+- Michael Gavin, Collin Jennings, Lauren Kersey, and Brad Pasanek. [Spaces of Meaning: Vector Semantics, Conceptual History, and Close Reading](https://dhdebates.gc.cuny.edu/read/untitled-f2acf72c-a469-49d8-be35-67f9ac1e3a60/section/4ce82b33-120f-423f-ba4c-40620913b305) explores how to use word embedding models to study concepts and their history.
+- Laura Nelson, [Leveraging the alignment between machine learning and intersectionality: Using word embeddings to measure intersectional experiences of the nineteenth century U.S. South](https://www.sciencedirect.com/science/article/pii/S0304422X21000115?dgcid=author) considers ways of using word embedding models in deliberately intersectional research applications, working through an example project on racialized and gendered identities in the nineteenth-century U.S. South.
+- Anouk Lang, [Spatial Dialectics: Pursuing Geospatial Imaginaries with Word Embedding Models and Mapping](https://modernismmodernity.org/forums/posts/spatial-dialectics) uses word embedding models to explore 'spatial imaginaries' in early twentieth-century Canadian periodicals, looking at gendered discourse and also at ways to combine results word embedding models with geospatial analysis.
+- Siobhan Grayson, Maria Mulvany, Karen Wade, Gerardine Meaney, and Derek Greene, [Novel2Vec: Characterising 19th Century Fiction via Word Embeddings](https://graysons.github.io/pdfs/grayson2016novel2vec.pdf) explores the use of word embedding models to study narrative formations in 19th-century novels.
 
 ## Acknowledgements
 
 We would like to thank Mark Algee-Hewitt and Julia Flanders for their contributions to various aspects of this lesson.
 
-## References
+## Endnotes
+
+[^1]: See, for example, work by [Benjamin Schmidt](http://bookworm.benschmidt.org/posts/2015-10-25-Word-Embeddings.html), [Ryan Heuser](https://ryanheuser.org/word-vectors/), and [Laura Nelson](https://doi.org/10.1016/j.poetic.2021.101539)
+
+[^2]: Indeed, Wevers and Koolen suggest that...... in Wevers, Melvin and Marijn Koolen. “Digital begriffsgeschichte: Tracing semantic change using word embeddings.” _Historical Methods: A Journal of Quantitative and Interdisciplinary History_ 53, no. 4 (2020), 226-243. https://doi.org/10.1080/01615440.2020.1760157
 
-Wevers, Melvin and Marijn Koolen. “Digital begriffsgeschichte: Tracing semantic change using word embeddings.” _Historical Methods: A Journal of Quantitative and Interdisciplinary History_ 53, no. 4 (2020), 226-243. https://doi.org/10.1080/01615440.2020.1760157
+[^3] for example Cordell (2017) and Rawson and Muñoz (2019) 
```

Comment:
I did not find any further information about these references in the original document. Would you be able to give a full citation for them? We use the Chicago referencing style in our lessons.